guides/source/active_storage_overview.md

```diff
@@ -141,19 +141,26 @@ gem "azure-storage", require: false
 
 ### Google Cloud Storage Service
 
+Add the [`google-cloud-storage`](https://github.com/GoogleCloudPlatform/google-cloud-ruby/tree/master/google-cloud-storage) gem to your `Gemfile`:
+
+```ruby
+gem "google-cloud-storage", "~> 1.11", require: false
```

Comment:
Agreed.

I will remove gem version dependency and update.

rgds

Ben

Reply:
OK i have removed version dependency and have posted for your review. pls see in the latest commit. if any issues etc. please let me know because I"m still learning how to contribute to rails.

rgds

Ben